### PR TITLE
[V2] Pick up extra aux_properties from event stream (moved from HASS).

### PR DESCRIPTION
**Pull request by shbatm**
There are several values for different nodes that do not appear in the `/rest/nodes/` call, but will be sent in the event stream once connected. Most notably, thermostat set-points and current operation modes. This change will pick these new properties up and store them in the aux_properties dict for the node.

Previously this was implemented in the `hacs-isy994` Home Assistant integration to allow setting state attributes, but this should be a core feature in PyISY because it applies to more than just the Home Assistant integration.

This also includes a change to use the `on level` (OL) as the hint if ON was called with no brightness. So if you have an on level of 60% and just call ON() with no brightness it will use a hint of 60%.

Other minor change: add 'formatted' value to aux properties

### CODE REVIEW

File: PyISY/constants.py [hunk 1]
```diff
@@ -399,3 +415,38 @@
     },
     "99": {"7": "on", "8": "auto"},  # Insteon Thermostat Fan Mode
 }
+
+INSTEON_RAMP_RATES = {
```
Comment (OverloadUT) [inline PyISY/constants.py]: Is this just here for consumers to import and use to understand how to interpret ramp rates in aux_properties?

It seems to me that we should change this to be a number representing the how many seconds the ramp rate is. That'll make working with the values easier (and more localizable)

Comment (shbatm) [reply PyISY/constants.py]: That was moved from the Home Assistant integration, I had left it there for a user to be able to translate where possible, but now I agree that it should actually be useful if we're going to include it.

I updated the aux_properties functions to translate the index to a true value in seconds.

Comment (shbatm) [reply PyISY/constants.py]: Thinking about more, should that be added as a separate property to not break anyone's integration (looking for and translating "RR" themselves? Or should we just document it as a breaking change?

Comment (OverloadUT) [reply PyISY/constants.py]: I think we should document it as a breaking change, but open to disagreement :)

Comment (shbatm) [reply PyISY/constants.py]: Well, I guess, considering RR is not reported in the initial query, and it was never stored in aux properties before, and I was the one that implemented it on the HA end to store a translated version... I'd say we should be safe.

Thinking about it, it's probably not even a breaking change... The RR event can still be picked up separately from the event stream.

**Comment by OverloadUT** [comment]
Looks good to me now!

**Pull request merged**